Task02/Task02/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task02
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите длину стороны ромба:");
+            double a = double.Parse (Console.ReadLine());
+            double P = a * 4;
+
+            Console.WriteLine("Введите градусную меру угла между смежными сторонами ромба:");
+            double v = double.Parse (Console.ReadLine());
+            double v2 = Math.PI * v / 180;
+            double v1 = v2;
+            double S = Math.Sin(v1) * a * a;
```

Comment:
Нет смысла вводить еще одну переменную `v1`
```suggestion
            double S = Math.Sin(v2) * a * a;
```